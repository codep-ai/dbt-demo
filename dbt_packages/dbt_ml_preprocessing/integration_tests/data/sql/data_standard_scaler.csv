id_col,col_to_scale_1,col_to_scale_2
1,5,1
2,8,7
3,10,9
4,-5,9
5,4,30
6,10,-20
7,-4,2
8,0,1
9,2,1
