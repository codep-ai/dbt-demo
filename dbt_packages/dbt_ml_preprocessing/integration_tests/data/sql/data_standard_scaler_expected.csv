id_col,col_to_scale_1,col_to_scale_2,col_to_scale_1_scaled,col_to_scale_2_scaled
1,5,1,0.3162277660168379,-0.2822617512808164
2,8,7,0.8854377448471461,0.2094200090147992
3,10,9,1.2649110640673515,0.3733139291133377
4,-5,9,-1.5811388300841898,0.3733139291133377
5,4,30,0.12649110640673514,2.0942000901479925
6,10,-20,1.2649110640673515,-2.003147912315471
7,-4,2,-1.391402170474087,-0.2003147912315471
8,0,1,-0.6324555320336759,-0.2822617512808164
9,2,1,-0.2529822128134704,-0.2822617512808164
